src/ue9.py

```diff
@@ -275,7 +275,7 @@ def discoveryUDP(self):
         listen = True
         while listen:
             #We will wait 2 seconds for a response from a Ue9
-            rs,ws,es = select.select(inputs, [], [], 1)
+            rs, _, _ = select.select(inputs, [], [], 1)
```

Comment:
Changes like this one make the code less readable.  I don't think we should replaces descriptive identifiers with underscores to appease some linting tool.  FWIW, I have also a large number of unused variables in this codebase.